data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+			action
+				fail "Unfettered: Neutral With The Hai"
+
+mission "Unfettered: Untrustable"
+	to offer
+		has "reputation: Hai" < 0
+		has "Unfettered: Neutral With The Hai: done"
+		not "Unfettered: Neutral With The Hai: failed"
+	on offer
+		dialog `Since you have not respected your word to be neutral to the Hai you have received the agreed fine of 100 millions. Should you refuse to pay it the Unfettered will no longer consider you a friend.`
+	on accept
+		fine 100000000
+	on decline
+		"reputation: Hai (Unfettered)" <?= -1
+
+
+mission "Unfettered: Bring Hai"
+	name `Bring True Hai back`
+	description `Bring <bunks> Hai from <stopovers> to <origin>.`
+	source "Firelode"
+	stopover "Snowfeather"
+	passengers 10
+	blocked "You will need room for 10 passengers in order to take the next Unfettered mission."
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+		has "reputation: Hai" >= 0
+		or
+			not "Unfettered: offered bring hai"
+			random < 20
+	on defer
+		set "Unfettered: offered bring hai"
+	on offer
+		conversation
+			`Looking outside you see the sun is setting... it is a breathtaking view that reveals a big white structure in the distance. As you exit your ship you notice movement. You are partly carried by a crowd of Unfettered that seem to be heading to a big, yet simple, square-shape stadium nearby. That building is grandiose compared to its surroundings, distinguished by its sheer length and pure white appearance. In contrast, most buidlings around it appear to be bars or houses, but they are made from less pure materials, or maybe they just have not been cleaned in a long time. Something of great interest must be happening in there, though you notice some Unfettered are just going to the bars around, chatting.`
+			choice
+				`	(Go to the stadium.)`
+				`	(Ignore them and go back to my ship.)`
+					defer
+			`	As you head towards the temple, moving with the crowd, you can start to hear words being spoken loudly. A couple with 4 children is speaking, or more precisely, begging, "... and we fear the next season will be even more rough. We need foundings to restore the lands, nothing grows on them anymore!" The "Chiefs" speak together, and then grant her her request, before the next one in line comes. The sun is almost set, and you can see that the Chiefs are growing a bit impatient. "My cousins would like to join me, but they lack the funds to afford it. Would you allow more to join us, and become true?" This seems to bother them, and they answer briefly, "We will contact True ship captains to ask them if they can, but I fear they are indeed busy these days, which is why the prices have gone up." Then they rise up, and one speaks to the crowd, "It is late, you would do well to go home to care for those near you. Never hesitate to come back to us in the next cycle should you need help." At this, the crowd leaves, as do the Chiefs later on. You notice one of them looking at you, pensive, as he has just noticed your presence.`
```

Comment:
It could be called many ways, but perfectly looking white building in the sun inspires temple to me.
This is happening inside the structure, and the sound is amplified to make sure all can hear it (I'll specify it)
The crowd just keeps arriving, there was already a crowd there before the one that arrived; they don't want to spend all the time waiting, and some come just for the part after this; which is a "party" if you will.